packages/suite-data/src/message-system/config/config.v1.json

```diff
@@ -1,6 +1,51 @@
 {
     "version": 1,
-    "timestamp": "2021-05-01T00:00:00+00:00",
-    "sequence": 1,
-    "actions": []
+    "timestamp": "2021-06-09T12:00:00+00:00",
+    "sequence": 2,
+    "actions": [
+        {
+            "conditions": [
+                {
+                    "os": {
+                        "macos": "*",
+                        "linux": "!",
+                        "windows": "!",
+                        "android": "!",
+                        "ios": "!"
+                    },
+                    "environment": {
+                        "desktop": "!",
+                        "mobile": "!",
+                        "web": "*"
+                    },
+                    "browser": {
+                        "firefox": "!",
+                        "chrome": "91",
+                        "chromium": "!"
```

Comment:
chromium is not affected? i would say it might

Reply:
yep it is